src/components/Book.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styles from '../style/Header.module.css';
+
+// eslint-disable-next-line react/prop-types
```

Comment:
- [ ] Kindly ensure that you did not disable the linter (this was not there in the previous review), linters are there as a guide to pick issues earlier. I think this [material] might be of help. 🙏